com.vscodium.codium.desktop

```diff
@@ -0,0 +1,18 @@
+[Desktop Entry]
+Name=VSCodium
+Comment=Code Editing. Redefined.
+GenericName=Text Editor
+Exec=codium --no-sandbox --unity-launch --reuse-window %F
```

Comment:
You should use zypak instead of --no-sandbox.

Reply:
Note it's already included in the base app so you just need to change this line to use zypak-wrapper. Example app: https://github.com/flathub/com.microsoft.Teams

Reply:
hey! sorry for the late response recently at my job the amount work has increased. @bilelmoussaoui I applied your suggestion, thanks for the link @barthalion. Also I update package to the latest version of the Codium.